WooCommerce/WooCommerceTests/Authentication/StorePickerViewModelTests.swift

```diff
@@ -0,0 +1,198 @@
+import XCTest
+import Yosemite
+import protocol Storage.StorageManagerType
+@testable import WooCommerce
+
+final class StorePickerViewModelTests: XCTestCase {
+    private var storageManager: MockStorageManager!
+
+    override func setUp() {
+        super.setUp()
+        storageManager = MockStorageManager()
+    }
+
+    override func tearDown() {
+        super.tearDown()
```

Comment:
super nit: usually `super.tearDown()` is called the last, though it probably doesn't matter in this case

Reply:
Updated in 84f7e94faa1f2ea34d20cb1db382433bea63eb7a.